test/e2e/extra_test.go

```diff
@@ -0,0 +1,67 @@
+package e2e_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions"
+	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+var _ = Describe("Deploy cluster", Label("cluster-extra-ns"), func() {
```

Comment:
Do you want to rename the test itself?

Reply:
@zonorti 
Yes, but I'm not sure about the naming just wanted sth different to `advanced` because of its collision with `advancedDeployment`! any suggestions?